doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -78,18 +86,23 @@ command:
 
 include::../protos/vkGetDeviceProcAddr.txt[]
 
-  * pname:device is the logical device that provides the function pointer.
-  * pname:pName is the name of any Vulkan command whose first parameter
-    is one of
-  ** sname:VkDevice
-  ** sname:VkQueue
-  ** sname:VkCommandBuffer
-
-If pname:pName is not the name of one of these Vulkan commands, and is
-not the name of an extension command belonging to an extension enabled for
-pname:device, then fname:vkGetDeviceProcAddr will return `NULL`.
-
-include::../validity/protos/vkGetDeviceProcAddr.txt[]
+The table below defines the various use cases for fname:vkGetDeviceProcAddr
+and expected return value for each case.
+
+.vkGetDeviceProcAddr behavior
+[width="80%",options="header",align="center"]
+|=====
+| pname:device | pname:pName | return value
+| NULL | * | undefined
```

Comment:
I would be careful with the `NULL`. Are you sure implementations won't use 0 as the parameter? Besides, it already falls into the "invalid device" case below anyway.